src/Bundle/CollectionFieldBundle/Tests/CollectionFieldTypeTest.php

```diff
@@ -240,7 +240,7 @@ public function testWritingGraphQLData()
         $admin = new User();
         $admin->setRoles([User::ROLE_PLATFORM_ADMIN]);
         $this->container->get('security.token_storage')->setToken(
-          new UsernamePasswordToken($admin, null, 'main', $admin->getRoles())
+          new UsernamePasswordToken($admin, null, 'api', $admin->getRoles())
```

Comment:
Revert this change but send csrf_token with requests instead

Reply:
not fixed: created an issue for this #13